scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -232,36 +235,149 @@ def release_paths(self) -> List[Path]:
         return release_paths
 
 
+class ModelType(Enum):
+    """Enum for the Model Types accepted in ModelConfig."""
+
+    MLFLOW = "mlflow_model"
+    CUSTOM = "custom_model"
+    TRITON = "triton_model"
+
+
+class ModelFlavor(Enum):
+    """Enum for the Flavors accepted in ModelConfig."""
+
+    HFTRANSFORMERS = "hftransformers"

```

Comment:
>HFTRANSFORMERS = "hftransformers [](http://example.com/codeflow?start=4&length=32)

Could we add this in the azureml-asset repo instead?

Reply:
We should provide support for all the fields in model.yaml file. Removing these would mean we cannot handle 'flavor' field in model.yaml.